docs/prometheus.md

```diff
@@ -16,5 +16,7 @@ The Ingress Controller exports the following metrics:
   * `controller_nginx_last_reload_status`. Status of the last NGINX reload, 0 meaning down and 1 up.
   * `controller_nginx_last_reload_milliseconds`. Duration in milliseconds of the last NGINX reload.
   * `controller_ingress_resources_total`. Number of handled Ingress resources. This metric includes the label type, that groups the Ingress resources by their type (regular, [minion or master](./../examples/mergeable-ingress-types))
+  * `controller_virtualserver_resources_total`. Number of handled VirtualServer resources.
+  * `controller_virtualserverroute_resources_total`. Number of handled VirtualServerRoute resources.
```

Comment:
Should we add a clarification that the VSR needs to be referenced by a VS in order to appear in the metrics? Just in case users can be confused if they deploy a VSR and metrics are not updating. What do you think @Dean-Coakley , @pleshakov ?

Reply:
I will add a note. While I think familiar users should know what `handled` means. New users may be misled. Happy to revert the change if @pleshakov disagrees.